signature.py

```diff
@@ -0,0 +1,21 @@
+from eth_keys.datatypes import (
+    Signature,
+)
+
+from eth_utils import (
+    to_checksum_address,
+)
+
+from eth_keys.utils.address import (
+    public_key_bytes_to_address,
+)
```

Comment:
Does this import acts differently than this?
```py
from eth_keys.utils.address import public_key_bytes_to_address
```

Reply:
they are the same

Reply:
👍